.lintr

```diff
@@ -7,8 +7,8 @@ linters: linters_with_defaults(
     undesirable_function_linter = undesirable_function_linter()
   )
 exclusions: list(
-    "R/data.R" = Inf,
+    "R" = Inf,
     "inst" = list(undesirable_function_linter = Inf),
     "vignettes" = list(undesirable_function_linter = Inf),
-    "data-raw" = list(undesirable_function_linter = Inf)
+    "data-raw" = Inf
```

Comment:
I'm not sure if we should turn off lintr completely for all files in `R` and `data-raw`.

@manciniedoardo , what do you think?

Reply:
ideally not completely, is there a more granular approach?